wiki/Beatmap_Editor_Guides/Changing_the_Map's_Artist_or_Title/en.md

```diff
@@ -1,10 +1,13 @@
+Changing the Map's Artist or Title
+===================================
+
 [*Full Submission with Title/Artist Change -- Mistakes* by NoHitter (hijacked by peppy)](https://osu.ppy.sh/forum/t/24443)
 
 This is until osz2 will be implemented.
 
 When changing the **title or artist** in a map, a problem may occur in which you may not be able to submit it without making a duplicate map.
 
-<span style="color:#FF4000;">**DO NOT SUBMIT IF THE EDITOR "WEB" MENU IS DISABLED. YOUR MAP WILL DUPLICATE.**</span>
+**DO NOT SUBMIT IF THE EDITOR "WEB" MENU IS DISABLED. YOUR MAP WILL DUPLICATE.**
```

Comment:
no need to CAPS THIS ALL OUT, I GUESS..
`**Do not submit if the editor "web" menu is disabled. Your map will duplicate.**`